circle.py

```diff
@@ -33,6 +33,31 @@ def generate_2D_coordinate(x_min, x_max, y_min, y_max):
     return (x, y)
 
 
-# Below is an example of how to generate a random 2D coordinate
-x, y = generate_2D_coordinate(-1, 1, -1, 1)
+'''
+ @brief	    Function that calcultes pi
+ @details   The function randomly places points in a 1x1 box using generate_2D_coordinates. It uses the ratio of points that land in a circle with radius one vs the number of points thrown to calculate pi.
+ @param     num_darts the number of points to be tested
+ @return    The calculated value for pi	
+
+
+'''
+
+def calculate_pi(num_darts):
+	hits = 0
+	misses = 0
+	for i in xrange(int(num_darts)):	
```

Comment:
I'm a bit confused about the conversion to num_darts.  Do we ever expect to have function calls where num_darts is not an integer? If you want to enforce that the input be an integer, the better way to do that is raise an error where type(num_darts) is not an int.